.github/dot/specs.dot

```diff
@@ -50,6 +50,7 @@ graph {
         Monad -- Traversable;
 
         Monoid -- Category [style=dotted; arrowhead=none; constraint=false];
+        Semigroupoid -- Category
```

Comment:
Since there is `Monoid -- Category`, I suggest to add `Semigroup -- Semigroupoid`.

Reply:
Make sense since we already mention that Category generalizes Monoid in the documentation.